resources/views/dashboard/navbar.blade.php

```diff
@@ -29,6 +29,7 @@
                     @endif
                 @endfor
             </ol>
+            @show
```

Comment:
Shouldn't this be `$endsection`?

Reply:
No, `@show` will allow you to have default content if nothing is written to the section.
`@endsection` will try to write this content the `@yield('breadcrumbs')`